apps/web/public/static/locales/en/common.json

```diff
@@ -2439,5 +2439,8 @@
   "csat_score": "CSAT Score",
   "lockedSMS": "Locked SMS",
   "signing_up_terms": "By proceeding, you agree to our <1>Terms</1> and <2>Privacy Policy</2>.",
+  "unable_to_subscribe_to_the_platform": "Unable to subscribe to the platform, please try again later",
+  "updating_oauth_client_error": "There has been an error updating this OAuth client",
+  "creating_oauth_client_error": "There has been an error creating this OAuth client",
```

Comment:
`There has been an error updating the OAuth client`, `There has been an error creating the OAuth client` just some very minor details otherwise eveyrthing else looks good :)

Reply:
Yeah, that sounds good.